lib/mix/test/fixtures/no_source_code/mix.exs

```diff
@@ -0,0 +1,10 @@
+defmodule TestStale.MixProject do
```

Comment:
We don't need a new fixture, you can pick one of the existing ones, such as `compile_erlang`. :)